hikari/applications.py

```diff
@@ -659,9 +659,12 @@ class Application(guilds.PartialApplication):
     install_parameters: ApplicationInstallParameters | None = attrs.field(eq=False, hash=False, repr=False)
     """Settings for the application's default in-app authorization link, if enabled."""
 
-    approximate_guild_count: int = attrs.field(eq=False, hash=False, repr=False)
+    approximate_guild_count: int | None = attrs.field(eq=False, hash=False, repr=False)
     """The approximate number of guilds this application is part of."""
 
+    approximate_user_install_count: int | None = attrs.field(eq=False, hash=False, repr=False)
+    """The approximate number of users that have installed this application."""
+
```

Comment:
They can only be None for partial applications. For full applications they will always be there :)


```suggestion
    approximate_guild_count: int = attrs.field(eq=False, hash=False, repr=False)
     """The approximate number of guilds this application is part of."""
 
    approximate_user_install_count: int = attrs.field(eq=False, hash=False, repr=False)
    """The approximate number of users that have installed this application."""

```